appengine/users/README.md

```diff
@@ -0,0 +1,33 @@
+# Users Authentication sample for Google App Engine
+
+This sample demonstrates how to use the [Users API][appid] on [Google App
+Engine][ae-docs].
+
+[appid]: https://cloud.google.com/appengine/docs/java/users/
+[ae-docs]: https://cloud.google.com/appengine/docs/java/
+
+## Running locally 
+This example uses the
+[Maven gcloud plugin](https://cloud.google.com/appengine/docs/java/managed-vms/maven).
+To run this sample locally:
+
+    $ mvn gcloud:run
+
+## Deploying
+In the following command, replace YOUR-PROJECT-ID with your
+[Google Cloud Project ID](https://developers.google.com/console/help/new/#projectnumber).
+
+    $ mvn gcloud:deploy -Dgcloud.gcloud_project=YOUR-PROJECT-ID
```

Comment:
+1 Sorry, to lead you astray with https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/appengine/appidentity. I need to update that since the gcloud plugin is not explicitly supporting app engine standard.

It's actually `appengine:update` to do a deploy. The arguments look like:
```
mvn appengine:update -Dappengine.appId=java-docs-samples-swast-1 -Dappengine.version=20160612-urlfetch
```